IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetry.h

```diff
@@ -26,14 +26,13 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class MSIDRequestTelemetrySerializedItems;
+@class MSIDCurrentRequestTelemetrySerializedItem;
```

Comment:
We don't use it in the header, so we can remove it.

Reply:
Thanks for pointing that out. It's removed now.